this_is_the_way.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+
+def this_is_the_way():
+    res_lst = []
+    path = input("Enter path to directory: ")
```

Comment:
When you write code, you should aim to write it as "pure" a possible - avoid IO (input output) operations near logic. why is that? if you want to use this logic elsewhere - for example, receive a path from a place other than the standart input, you cant right now, receive the path (and the "deep") as parameters from the outside